HealthMonitoring.Persistence/EndpointStatsPersistCoordinator.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
+using System.Collections.Generic;
+using HealthMonitoring.Forwarders;
+using HealthMonitoring.Management.Core;
+using HealthMonitoring.Management.Core.Repositories;
+using HealthMonitoring.Model;
+
+namespace HealthMonitoring.Persistence
+{
+    public class EndpointStatsPersistCoordinator : IEndpointStatsPersistCoordinator

```

Comment:
I meant rather that in other scenarios, classes using repository could expect the dependency of IEndpointStatsRepository  and use it directly.

It would be good to hear other people thoughts here